src/components/UpcomingEventsList.js

```diff
@@ -82,15 +106,15 @@ const UpcomingEventsList = () => {
       {orderedUpcomingEvents?.map(
         ({ title, to, formattedDetails, date, location }, idx) => {
           return (
-            <EventListItem key={idx}>
-              <EventCard
-                title={title}
-                to={to}
-                date={date}
-                description={formattedDetails}
-                location={location}
-              />
-            </EventListItem>
+            <EventCard
+              key={idx}
+              title={title}
+              to={to}
+              date={date}
+              description={formattedDetails}
+              location={location}
+              idx={idx}
```

Comment:
If we go with the approach on `EventCard` of passing `isEven` as a prop, you could calculate that here, naming it the same to make it easy to track, and then you could remove needing to declare this as a constant in the EventCard component. 

 ```suggestion
              isEven={idx % 2 === 0}
```